rego.json

```diff
@@ -0,0 +1,100 @@
+{
+    "policy template": {
+        "prefix": [
```

Comment:
suggestion: since the templates are Regal specific, rather than for any Rego policy, it might make sense to have that as the matching prefixes? We seem to have this in the other templates.

Reply:
So simply include "regal" as prefix for their names?